packages/helpermodules/hardware_configuration.py

```diff
@@ -16,5 +17,13 @@ def get_hardware_configuration_setting(name: str):
         return json.loads(f.read())[name]
 
 
+def get_serial_number() -> str:
+    try:
+        with open("/home/openwb/snnumber", "w") as file:
+            return file.read().replace("\n", "")
+    except io.UnsupportedOperation:
```

Comment:
Notwendige Anpassung, wenn der Import angepasst wird:
```suggestion
    except UnsupportedOperation:
```